<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gams::studio::modelinspector::ModelInspector</class>
 <widget class="QWidget" name="gams::studio::modelinspector::ModelInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="SectionTreeView" name="sectionView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="statisticPage1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="StatisticsWidget" name="statisticWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="fullView">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="gams::studio::modelinspector::ModelInstanceTableView" name="modelInstanceView">
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatisticsWidget</class>
   <extends>QWidget</extends>
   <header>modelinspector/statisticswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SectionTreeView</class>
   <extends>QTreeView</extends>
   <header>modelinspector/sectiontreeview.h</header>
  </customwidget>
  <customwidget>
   <class>gams::studio::modelinspector::ModelInstanceTableView</class>
   <extends>QTableView</extends>
   <header>modelinspector/modelinstancetableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
